cumulusci/core/dependencies/utils.py

```diff
@@ -1,4 +1,13 @@
-from collections import namedtuple
+from logging import Logger
+from typing import NamedTuple
+
+from cumulusci.core.config.org_config import OrgConfig
+from cumulusci.core.config.project_config import BaseProjectConfig
 
 # data structure mimicking a task for use with the metadata API classes
-TaskContext = namedtuple("TaskContext", ["org_config", "project_config", "logger"])
```

Comment:
@TheBitShepherd  Can these hard coded values be parameterized in to some properties file ?